public/toolkit/styles/atoms/_tables.scss

```diff
@@ -569,6 +574,10 @@ table {
   }
 }
 
+.table-cell-units {
+  width: 100px;
```

Comment:
why fix it at a number of pixels? I'm not sure that's something we've done before.

Reply:
@akegan I attempted to explain this in my PR comment but it's confusing so I'll try and lay it out in a more detailed way:

Right now the width of the first column is determined by the width of the longest text in the column, which is basically always "available". So the current version is essentially fixed width but just out of luck.

So say we have a listing with two apt types in it, both tables are aligned:
```
--- 1BR table ---
1        |
available|
...      |
waitlist |
...      |
3        |
available|

--- 2br table ---
3        |
available|
```

This is because in the current version, the `100% AMI`/`Moderate Income` labels aren't actually part of the table column, they're absolutely positioned on top.

This fix moves that text into the columned themselves, and the above column now looks like this (notice how the two tables have different column widths now):

```
--- 1BR table ---
Low Income     |
1              |
available      |
...            |
Moderate Income|
waitlist       |
...            |
High Income    |
3              |
available      |
--- 2br table ---
Low Income|
3         |
available |
```

Here's a screenshot of what it looks like without the width constraint:
<img src="https://user-images.githubusercontent.com/64036574/85327775-61efa700-b484-11ea-8204-d44d78bca56d.png" width="530" />

But a fixed pixel width isn't the only solution, we could do any of the following:
1. Use fixed pixel width
2. Use percent-based fixed width (units col takes up x% of table, would need to be different for sale and rental tables)
3. Don't use fixed width at all and accept that separate tables will not be aligned

What do you think?

Reply:
hm, I guess it looks ok! maybe once chris is back we can get a sense for him what he would choose and/or what pixel width he thinks makes sense